features/abortable-fetch.yml.dist

```
@@ -13,3 +13,18 @@ status:
     firefox_android: "57"
     safari: "12.1"
     safari_ios: "12.2"
+compat_features:
+  # baseline: high
+  # baseline_low_date: 2018-04-17
+  # baseline_high_date: 2020-10-17
+  # support:
+  #   chrome: "66"
+  #   chrome_android: "66"
+  #   edge: "16"
+  #   firefox: "57"
+  #   firefox_android: "57"
+  #   safari: "11.1"
+  #   safari_ios: "11.3"
+  - api.fetch.init_signal_parameter
+  # Same status as overall feature
+  - api.Request.signal
```

White space to separate "generations" of keys would help readability, I think

```suggestion
  #   safari_ios: "11.3"
  - api.fetch.init_signal_parameter

  # Same status as overall feature
  - api.Request.signal
```

---

I agree, but wasn't sure how to achieve it. Do you know?

---

I found a way!